sim/lib/data/queries_sf/office.txt

```diff
@@ -1,5 +1,3 @@
 node["office"]
-node["shop"]
```

Comment:
In the past, to solve the problem that most people work in a small number of offices, we also set various types of shops as "offices". Now we need these shops to be retail stores, so I removed it from here.